test/libsolidity/syntaxTests/tupleAssignments/assignments_to_tuple_and_non_tuple_expressions_of_tuple_types.sol

```diff
@@ -0,0 +1,34 @@
+contract C {
+	uint[] public array;
+
+	function f() public {
+		(f()) = (); // Function call resolving to an empty tuple expression of type tuple(). Error
+	}
```

Comment:
Well, this is still not quite right. The point of these comments was to point out where the whole expression on the left-hand side is a tuple expression.

```suggestion
	function f() public {
	    // (f()) is not a tuple expression, but its value is a tuple.
		(f()) = ();
	}
```

Reply:
This was just an example. This applies to all the comments in the test.

Reply:
I know, I updated the others as well ;)